package.json

```diff
@@ -21,42 +21,40 @@
   },
   "scripts": {
     "build": "yarn build:components",
-    "build:all": "run-s build:preview build:site",
-    "postbuild:all": "yarn mv:preview",
-    "build:clean": "run-s clean build",
-    "build:components": "lerna run --scope \"${SCOPE:-@spectrum-css/*}\" --ignore \"@spectrum-css/{*-builder*,preview,generator}\" build",
-    "postbuild:components": "yarn docs:mod",
-    "build:preview": "nx build @spectrum-css/preview",
-    "build:site": "gulp buildDocs -LLL",
-    "cache:clean": "nx reset",
+    "build:all": "run-s build:preview build:site mv:preview",
+    "build:components": "nx run-many --all --exclude \"@spectrum-css/{*-builder*,preview,generator}\" --target build",
+    "postbuild:components": "node tasks/mod-extractor.js && prettier --cache --write components/*/metadata/*.md > /dev/null",
+    "build:preview": "nx run @spectrum-css/preview:build",
+    "build:site": "gulp build -LLL",
     "ci": "yarn build:all",
-    "ci:storybook": "nx storybook @spectrum-css/preview",
-    "clean": "yarn cache:clean && run-p clean:*",
-    "clean:components": "lerna exec --scope \"${SCOPE:-@spectrum-css/*}\" --ignore \"@spectrum-css/{*-builder*,preview,generator}\" -- rimraf dist",
-    "clean:docs": "rimraf dist",
-    "clean:preview": "nx clean @spectrum-css/preview",
-    "predev": "yarn build:components",
-    "dev": "NODE_ENV=development BROWSERSYNC_OPEN=true gulp dev",
-    "docs:mod": "node tasks/mod-extractor.js",
-    "postdocs:mod": "prettier --cache --write components/*/metadata/*.md > /dev/null",
-    "lint:components": "node ./tasks/packageLint.js",
-    "lint:styles": "stylelint",
-    "mv:preview": "rimraf dist/preview && mv tools/preview/storybook-static dist/preview",
+    "ci:storybook": "nx run @spectrum-css/preview:storybook",
+    "clean": "run-p clean:*",
+    "clean:cache": "nx reset",
+    "clean:components": "rimraf components/*/dist",
+    "clean:preview": "nx run @spectrum-css/preview:clean",
+    "clean:site": "rimraf dist",
+    "dev": "npm-run-all -s build:components -p dev:*",
```

Comment:
Running this command works well for serving the docs site, but I'm noticing that when I click over to view the Storybook instance, it looks like some of the styles are missing

<img width="726" alt="Screenshot 2023-08-22 at 1 47 02 PM" src="https://github.com/adobe/spectrum-css/assets/360251/4f2777e0-0760-4541-9440-9e79e9c7a55c">